File: Basic/Loop/219432G.c
#include <stdio.h>
#define intll long long

void solution() {

	int n;
	scanf("%d", &n);

	intll fact = 1;
	for (int i = 1; i <= n; i++) {
		fact *= i;
	}

	printf("%lld\n", fact);
}

int main() {
	// char TEST_CASE = !1;
	int t = 1;
	// if (TEST_CASE == 1) {
	// 	scanf("%d", &t);
	// }

	while (t--) {
		solution();
	}
	return 0;
}

File: Basic/Loop/219432G.go
package main

import (
	"fmt"
)

func solution() {

	var n int
	fmt.Scan(&n)

	fact := 1
	for i := 1; i <= n; i++ {
		fact *= i
	}

	fmt.Println(fact)
}

func main() {
	TEST_CASE := true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}
